### DISCUSSION TITLE
how can i resolve the copilot extension error about login?

### DISCUSSION THREAD

**Discussion by huanghuixin1** [eclipse-theia/theia#12728] [Q&A]
When I install the Copilot plugin and am unable to log in, an error will be reported as follows:
[ERROR] [default] [2023-07-19T02:13:47.479Z] GitHub Copilot could not connect to server. Extension activation failed: "Invalid scheme 'theia'"

What is the solution to this problem

**Comment by msujew** [comment]
@huanghuixin1 I believe this issue (https://github.com/eclipse-theia/theia/issues/12821) might be related to the problem you're experiencing.
